Makefile

```diff
@@ -6,11 +6,40 @@
 #
 # Dev tools binaries and options
 #
+
+# Get rgbasm version
+ASMVER    := $(shell rgbasm --version | cut -f2 -dv)
+ASMVERMAJ := $(shell echo $(ASMVER) | cut -f1 -d.)
+ASMVERMIN := $(shell echo $(ASMVER) | cut -f2 -d.)
```

Comment:
Nitpick: if this check is moved after the `ASM := rgbasm` line, it may allow to use the `$(ASM)` var instead of hardcoding rgbasm.

(Not that anybody will use a different assembler though…)